cypress/integration/default/preview.spec.ts

```diff
@@ -5,17 +5,15 @@ describe('Preview Mode', () => {
     cy.findByText('Is preview? No')
 
     // enter preview mode
-    cy.request('/api/enterPreview').then(
-  (response) => {
-    expect(response.body).to.have.property('name', 'preview mode')
-  }
-)
-  cy.visit('/previewTest')
-  cy.findByText('Is preview? Yes!')
+    cy.request('/api/enterPreview').then((response) => {
+      expect(response.body).to.have.property('name', 'preview mode')
+    })
+    cy.visit('/previewTest')
+    cy.findByText('Is preview? Yes!', { selector: 'h1' })
 
-  // exit preview mode
-  cy.request('/api/exitPreview')
-  cy.visit('/previewTest')
-  cy.findByText('Is preview? No')
+    // exit preview mode
+    cy.request('/api/exitPreview')
+    cy.visit('/previewTest')
+    cy.findByText('Is preview? No', { selector: 'h1' })
```

Comment:
Why is Cypress in CI saying there is more than one instance of this text :confused-capybara:?

Reply:
I was baffled for ages, but if you look at the error in the dashboard, it seems there's a `<p>`, which is the route announcer (for screen readers)! No idea why it only shows for this route though.